gitman/git.py

```diff
@@ -47,12 +48,21 @@ def clone(repo, path, *, cache=settings.CACHE, sparse_paths=None, rev=None):
         git('clone', '--reference', reference, repo, os.path.normpath(path))
 
 
+def is_sha(rev):
+    """Heuristically determine whether a revision corresponds to a commit SHA.
+
+    Any sequence of 7 to 40 hexadecimal digits will be recognized as a
+    commit SHA. The minimum of 7 digits is not an arbitrary choice, it
+    is the default length for short SHAs in Git.
+    """
+    re.match('^[0-9a-f]{7,40}$', rev) is not None
```

Comment:
You need a `return` statement here. 😉

Reply:
Thanks for pointing it out. Too bad that Pylint did not warn me about that one. I mostly code in programming languages where no `return` statement is needed... so it's easy for me to forget!

Reply:
`pylint` _should_ catch that, but there was some sort of issue installing dependencies on Travis CI. Please rebase on `develop`.

Reply:
OK, it is probably because my Pylint is for Python _2.x_ and this project uses Python 3. I have rebased, let's see now.